course/en/chapter13/grpo_finetune.ipynb

```diff
@@ -0,0 +1,175 @@
+{
```

Comment:
<p>Do you think it makes sense to add yet another cell showcasing the inference snippet but this time directly via the hub via a pipeline?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>